cli/src/program.rs

```diff
@@ -236,6 +242,23 @@ impl ProgramSubCommands for App<'_, '_> {
                                     "Use the designated program id even if the account already \
                                      holds a large balance of SOL",
                                 ),
+                        )
+                        .arg(
+                            Arg::with_name("compute_unit_price")
+                                .long("compute-unit-price")
+                                .takes_value(true)
+                                .help(
+                                    "Set computeUnitPrice (tx priotity fee) to boost transactions resulting in faster execution speed \
+                                    [Note: priority fee is added in LAMPORTS, so enter priority fee in terms of LAMPORTS]",
+                                ),
+                        ).
+                        arg(
+                            Arg::with_name("max_retires")
+                                .long("max-retries")
+                                .takes_value(true)
+                                .help(
+                                    "Specifies the max retry count for failed transactions",
+                                ),
```

Comment:
these should be added in separate PRs

Reply:
so compute_unit_price is 1 pr and max_retries is another pr ?

Reply:
resolved